robyn/router.py

```diff
@@ -136,13 +136,11 @@ def add_middleware(
         def inner(handler):
             @wraps(handler)
             async def async_inner_handler(*args):
-                await handler(*args)
-                return args
+                return await handler(*args)
 
             @wraps(handler)
             def inner_handler(*args):
-                handler(*args)
-                return args
+                return handler(*args)
```

Comment:
This fixes a bug we never noticed in our tests, were a middleware would not return what it specifies after the `return` keyword, but its own arguments. As we always only modified the arguments in the middleware and returned them it was left unnoticed. It took me sometime to figure it out ^^

Reply:
Wow! Thank you for catching this. ❤️ 

I can imagine how annoying this would've been.

Reply:
Thanks that wasn't easy to find ^^